core/put/source/dev_ui.h

```diff
@@ -65,7 +65,7 @@ namespace put
         }
         typedef e_ui_shader::ui_shader_t ui_shader;
 
-        bool        init();
+        bool        init(ImGuiStyle &style);
```

Comment:
I think potentially it might be better to change this to a pointer, which is default to nullptr. If it is set to nullptr the default theme is used.

Reply:
actually after thinking about it, it doesn't need to be nullptr, you could do something like:

```c++
bool init(ImGuiStyle &style = default_pmtech_style())
```

And make a function which keeps the old defaults I set up.